src/Microsoft.AspNetCore.Mvc.Razor/RazorPageActivator.cs

```diff
@@ -17,6 +18,10 @@ namespace Microsoft.AspNetCore.Mvc.Razor
     /// <inheritdoc />
     public class RazorPageActivator : IRazorPageActivator
     {
+        // Delegates to create ViewDataDictionary of TModel
```

Comment:
This comment doesn't explain enough to be useful. Either explain **why** this is special or just remove the comment.